doc/iterations/2025.2/add_impact_analysis_before_tool_execution.adoc

```diff
@@ -0,0 +1,41 @@
+= Add Impact Analysis Before Tool Execution
+
+== Problem
+
+Some tool executions could have a significant impact on the model, and the specifier may want to warn the end user of this impact before validating the execution.
+
+== Key Result
+
+The specifier can declare that an impact analysis will be triggered before the execution of a tool.
+When a user triggers this tool, they receive initial feedback on the changes that will occur after the tool execution.
+The user can then decide whether to apply the tool or not.
+
+== Scenario
+
+=== Tool Execution on a Representation
+
+On a representation, a user executes a tool with an active impact analysis.
+A modal window opens, displaying the differences in the model before and after the tool execution.
+At this point, the tool execution is not persisted (and thus other users on the same representation are not notified that a tool has been triggered).
+The user can now choose to apply the tool or not.
+
+*  If the user cancels the tool execution, no changes are applied.
+*  If the user approves the tool execution, changes are applied to the representation for all users on it.
+
+== Breadboarding
+
+A modal window with a report listing the differences brought by the tool.
+
+== Cutting Backs
+
+For a first version, only tool on diagram will be concern.
```

Comment:
I'll just mention that things should still be developed in a generic manner to be applied easily later to other representations